ios/VisionTestApp/VisionTestApp/ModelRunner.mm

```diff
@@ -33,7 +33,7 @@ + (NSString *)run {
   auto img_tensor = torch::ones({3, 224, 224}, at::ScalarType::Float);
   inputs.push_back(c10::List<at::Tensor>(img_tensor));
   torch::autograd::AutoGradMode guard(false);
-  at::AutoNonVariableTypeMode nonVarTypeModeGuard(true);
+  at::AutoDispatchBelowADInplaceOrView nonVarTypeModeGuard(true);
```

Comment:
According to https://github.com/pytorch/pytorch/blob/master/docs/cpp/source/notes/inference_mode.rst#migration-guide-from-autononvariabletypemode, this would need to be replaced by `InferenceMode` I believe

Reply:
Good point. I'll replace it. Currently also debugging the build issue on Android.